src/style.css

```diff
@@ -0,0 +1,34 @@
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.title1 {
+  font-weight: bold;
+  font-size: 17px;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.articleN {
+  font-weight: normal;
+  font-size: 15px;
+  margin: 1em 0;
+}
+
+.list {
+  font-size: 13px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.parag {
+  font-size: 13px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+a {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice